lib/config/error-codes/services.json

```diff
@@ -238,6 +238,18 @@
           "code": 42,
           "message": "The collection name \"%s\" is invalid",
           "class": "BadRequestError"
+        },
+        "too_many_scroll": {
+          "description": "The maximum number of concurrent scroll requests as been reached. (See config.services.storageEngine.maxScroll)",
+          "code": 44,
+          "message": "Maximum number of concurrent scroll requests reached. (%s)",
```

Comment:
We shouldn't leak the configuration value for the maximum number of concurrent scroll requests: if  administrators want that information they can check the configuration files, normal users don't need it, and malicious users might find that information useful.